### ISSUE TITLE
Is there any processing logic related to NaN?

### ISSUE THREAD

**Issue by zhaosonggo** [stig/json-framework#294]

**Comment by stig** [comment]
Yes. The writer detects NaNs and errors here: https://github.com/SBJson/SBJson/blob/e36f22528daecab6f5803edfe271d6462768bd67/Classes/SBJson5StreamWriter.m#L471-L474

For the _parser_ there really is no special handling: there's no way to represent NaN in the JSON grammar, and this parser implements the grammar accurately. Loss of floating-point precision for big numbers is the only issue I am aware of.
